example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -49,10 +52,22 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // Set this value in your local.properties
-        if (BuildConfig.BACKTRACE_SUBMISSION_URL != null) {
-            backtraceClient = initializeBacktrace(BuildConfig.BACKTRACE_SUBMISSION_URL);
+        if (BACKTRACE_SUBMISSION_URL != null) {
+            backtraceClient = initializeBacktrace(BACKTRACE_SUBMISSION_URL);
         }
 
+        // Crash Loop Detector example
+        boolean isCLSafeModeReq = BacktraceDatabase.IsSafeModeRequired();
+        int crashesCount = BacktraceDatabase.ConsecutiveCrashesCount();
+        Log.i("BacktraceAndroid", "ConsecutiveCrashesCount: " + crashesCount);
+
+        View viewBackground = findViewById(R.id.viewBackground);
+        if(viewBackground != null) {
```

Comment:
fix formatting

Reply:
I don't think the formatting has been fixed. Can we run formatted?

Reply:
`if(` for example - but it would be better if we format the whole file - in case of some other formatting issues that we don't see